projects/admin/src/app/service/items.service.ts

```diff
@@ -28,6 +28,11 @@ import { UserService } from './user.service';
 })
 export class ItemsService {
 
+  /**
+   * Informs parent component to update item
+   */
+  updateItem$ = new Subject();
```

Comment:
I would not use a service for that but an Input/Output mechanism.

Reply:
Done